seqr/views/utils/file_utils.py

```diff
@@ -86,25 +90,28 @@ def save_uploaded_file(request, process_records=None):
         raise ValueError("Received %s files instead of 1" % len(request.FILES))
 
     # parse file
-    stream = request.FILES.values()[0]
+    stream = next(iter(request.FILES.values()))
     filename = stream._name
 
+    if not filename.endswith('.xls') and not filename.endswith('.xlsx'):
+        stream = TextIOWrapper(stream.file, encoding = 'utf-8')
+
     json_records = parse_file(filename, stream)
     if process_records:
         json_records = process_records(json_records, filename=filename)
 
     # save json to temporary file
-    uploaded_file_id = hashlib.md5(str(json_records)).hexdigest()
+    uploaded_file_id = hashlib.md5(str(json_records).encode('utf-8')).hexdigest()
```

Comment:
Codacy found an issue: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5795824)

Reply:
not relevant (this didn't actually change)